scripts/e2e-test.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -eu
+
+SCRIPTDIR=$(dirname "$(realpath "$0")")
+EXITV_INVALID_OPTION=1
+
+help() {
+  cat <<EOF
+Usage: $0 [OPTIONS]
+
+Options:
+  --testplan                 Test plan[lvm, zfs, hostpath, selfci etc.]
+Examples:
+  $0 --testplan lvm
+EOF
+}
+
+# Parse arguments
+while [ "$#" -gt 0 ]; do
+  case "$1" in
+    -T|--testplan)
+      shift
+      testplan="$1"
+      ;;
+    *)
+      echo "Unknown option: $1"
+      help
+      exit $EXITV_INVALID_OPTION
+      ;;
+  esac
+  shift
+done
+
+echo "Testplan: $testplan"
+# Get the array elements from the command-line argument
+array_str=$(python3 $SCRIPTDIR/testlists.py --testplan $testplan --install)
```

Comment:
Hmmm.. would it be simpler to have the python script invoke exec-tests.sh?
Then we wouldn't have to the split string into an array and iterate. (which IMO is fragile)
That would be more robust.
maybe for later.

Reply:
yeah, it can be done by python script which will be more robust.
Will take this as improvements in new PR